packages/blocks/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 6.0.5 (Unreleased)
```

Comment:
Under SemVer, new functionality qualifies as a minor patch bump. Patch bumps are reserved for bug fixes.

https://semver.org/

```suggestion
## 6.1.0 (Unreleased)
```